packages/barrier/SPECS/barrier.spec

```diff
@@ -0,0 +1,138 @@
+%global icon_path %{_datadir}/icons/hicolor/scalable/apps/%{name}.svg
+Summary: Use a single keyboard and mouse to control multiple computers
+Name: barrier
+Version: v2.3.2_51_g2d2e9298
+Release: 2%{?dist}
+License: GPLv2
+URL: https://github.com/debauchee/barrier/wiki
+Source0:
```

Comment:
Admiral, a rogue newline has infiltrated our Source0.

Reply:
Agh. Last time I swap between gvim and nedit for editing.